src/App.js

```diff
@@ -10,10 +11,10 @@ function App() {
     <div className="App">
       {/* <header className="App-header"></header> */}
       <Routes>
-        {/* <Route path="/" element={} /> */}
-      <Route path="/reviews" element={<Reviews/>} />
-      <Route path="/categories" element={<Categories/>} />
-      <Route path="/category/:slug" element={<Category/>} />
+        <Route path="/" element={ <Homepage /> } />
+        <Route path="/reviews" element={ <Reviews /> } />
+        <Route path="/categories" element={ <Categories /> } />
+        <Route path="/category/:slug" element={ <Category /> } />
```

Comment:
Ideally line 17 would use the reviews component from line 15. 
they are the same thing, except one is displaying filtered reviews, and the other displays them all, the only difference is the amount of data they display